src/modules/Dashboard/features/Respondents/Popups/EditRespondentPopup/EditRespondentPopup.test.tsx

```diff
@@ -0,0 +1,54 @@
+import { waitFor, screen, fireEvent } from '@testing-library/react';
+import axios from 'axios';
+
+import { renderWithProviders } from 'shared/utils/renderWithProviders';
+
+import { EditRespondentPopup } from '.';
+
+const onCloseMock = jest.fn();
+const successFakeRequest = () => new Promise((res) => res(null));
+
+const chosenAppletData = {
+  appletId: '12312',
+  respondentSecretId: '12312',
+  respondentId: '12312',
+  respondentNickname: 'respondentNickname',
+  ownerId: '1',
+};
+
+const commonProps = {
+  onClose: onCloseMock,
+  popupVisible: true,
+  chosenAppletData,
+};
+
+describe('EditRespondentPopup component tests', () => {
+  const mockedAxios = axios.create();
+
+  afterEach(() => {
+    jest.restoreAllMocks();
+  });
+
+  test('EditRespondentPopup should appear with respondentSecretId and respondentNickname', async () => {
```

Comment:
please don't use `EditRespondentPopup` in each test, it already has declaration in `describe` method